shuffle/cli/src/account.rs

```diff
@@ -143,7 +144,12 @@ pub async fn get_treasury_account(
     client: &DevApiClient,
     root_key_path: &Path,
 ) -> Result<LocalAccount> {
-    let treasury_account_key = load_key(root_key_path);
+    let treasury_account_key = match root_key_path.exists() {
+        true => Ok(load_key(root_key_path)),
+        false => Err(anyhow!(
+            "A node hasn't been created yet! Run shuffle node first"
+        )),
+    }?;
```

Comment:
let s = if condition {
    "first"
} else {
    "second"
};

Just fyi this works too for a true false check since match is a bit awkward to use for boolean matching

Reply:
Ahh yes. I used a match statement here because I thought that was more idiomatic to Rust and I've gotten so many comments in the past about using match statements over if/else blocks. But in a situation where I'm doing boolean matching, is the if/else considered idiomatic?

Reply:
Usually match is better but for boolean i find if else more readable -- it's just your preference and either way works